### ISSUE TITLE
multiple messages files

### ISSUE THREAD

**Issue by adros** [rxaviers/globalize-webpack-plugin#11]
As I understand from the doc / config, the idea is to have only one messages file (per locale) per whole application? Or is it possible to define multiple files with localization resources?

E.g. I would like to have

    moduleA/messages/en.json
    moduleA/messages/es.json
    moduleB/messages/en.json
    moduleB/messages/es.json

**Comment by rxaviers** [comment]
Correct, the current API supports one message file per whole application, but it should be easy technically to extend support for multiple messages files. I'm open to accept a PR with such addition, would you be willing to submit it? If so, please first add new comments with proposing changes to the API (e.g., how is config going to look like).

Thanks
